### PR TITLE
Add pandas and columsrow in reload

### PR DESCRIPTION
**Pull request by Adrien-Auclair**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Data
 numpy==1.10.1
```
Comment (achauve) [inline requirements.txt]: also update numpy to 1.10.4?

File: serenytics/source.py [hunk 2]
```diff
@@ -46,6 +47,44 @@ def reload_data(self, new_data):
...
+        for r in rows:
+            new_row = {}
+            for idx, column in enumerate(columns):
+                column_name = column if column != '$$count$$' else '__count__'
+                new_row[column_name] = r[idx]
+            new_data.append(new_row)
+
+        self.reload_data(new_data)
+
+    def reload_data_from_pandas_dataframe(self, df):
```
Comment (achauve) [inline serenytics/source.py]: rename to `reload_data_from_dataframe` without pandas?

File: serenytics/source.py [hunk 3]
```diff
@@ -129,3 +168,6 @@ def columns(self):
     @property
     def rows(self):
         return self._raw_data.get('data', [])
+
+    def get_as_pandas_dataframe(self):
```
Comment (achauve) [inline serenytics/source.py]: rename to `get_as_dataframe`?

**Comment by achauve** [comment]
Minor comments, and then ok for merge

File: requirements.txt (hunk 1 above)
Comment (Adrien-Auclair) [inline requirements.txt]: yes

File: serenytics/source.py (hunk 3 above)
Comment (Adrien-Auclair) [inline serenytics/source.py]: yes

File: serenytics/source.py (hunk 2 above)
Comment (Adrien-Auclair) [inline serenytics/source.py]: yes

**Pull request merged**